.circleci/config.yml

```diff
@@ -93,6 +93,15 @@ jobs:
       - run: sudo apt-get install -y libxml2-utils
       - *install_tox
       - run: tox -e validate_test_tools
+  test_galaxy_release:
```

Comment:
@natefoo Did you also mean to add this job to the `get_code_and_test` workflow? Because it's unused at the moment.

Reply:
Probably? Thanks for noticing this.

Reply:
Cool, I will add it to https://github.com/galaxyproject/galaxy/pull/13035

Reply:
Thank you!